### ISSUE TITLE
missing permissions for operator

### ISSUE THREAD

**Issue by davidkarlsen** [zalando/postgres-operator#1335]
Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?** e.g.
registry.opensource.zalan.do/acid/postgres-operator:v1.6.0
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?** [AWS K8s | GCP ... | Bare Metal K8s]
baremetal/openshift
- **Are you running Postgres Operator in production?** [yes | no]
no, blocked by issues like this
- **Type of issue?** [Bug report, question, feature request, etc.]
bug report

the operator misses this permission:
```
time="2021-01-27T16:33:40Z" level=warning msg="master is not running, generated master endpoint does not contain any addresses" cluster-name=anchore/anchore-cluster pkg=cluster worker=0
time="2021-01-27T16:33:40Z" level=error msg="could not create cluster: could not create master endpoint: could not create master endpoint: endpoints is forbidden: User \"system:serviceaccount:postgres-operator:postgres-operator\" cannot create resource \"endpoints\" in API group \"\" in the namespace \"anchore\"" cluster-name=anchore/anchore-cluster pkg=controller worker=0
[et2448@Davids-Work-MacBook-Pro anchore (⎈ |fluxcd/api-os-dts-global-finods-com:6443/david.karlsen@evry.com:anchore)]$ k create rolebinding admin --clus
```

I installed the operator using this chart:
```
repository: https://raw.githubusercontent.com/zalando/postgres-operator/master/charts/postgres-operator/
    name: postgres-operator
    version: 1.6.0
```

**Comment by FxKu** [comment]
I see that you've already found other threads (#985) on this topic. For openshift you have to adjust some options in the `values.yaml` file, I guess. At least [`kubernetes_use_configmaps`](https://github.com/zalando/postgres-operator/blob/9d5841cdaaefba9aba1a12112ebca9b7dee00eb4/charts/postgres-operator/values.yaml#L36) should be `true` as there are no endpoints in OpenShift. The will then create the right RBAC for you.

**Comment by FxKu** [comment]
Closing this issue since permissions were fixed in #1760.
